apps/web-portal-client/lib/components/Popover.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {useState} from 'react'
+import PropTypes from 'prop-types'
+import Tippy from '@tippy.js/react'
+import 'tippy.js/dist/tippy.css'
+import 'tippy.js/themes/light-border.css'
+
+import {childrenPropTypes} from '../utils/childrenPropTypes'
+
+export function Popover({title, text, children}) {
+  const [visible, setVisible] = useState(false)
+
+  const content = (
+    <div>
+      <div className="text-black opacity-75 font-semibold p-3 mb-0 border-b border-solid border-gray-200 uppercase rounded-t-lg">
+        {title}
+      </div>
+      <div className="text-black p-3">{text}</div>
+    </div>
+  )
+
+  return (
+    <Tippy
+      content={content}
+      visible={visible}
+      arrow
+      animation="scale"
+      duration={0}
```

Comment:
Another way to write it is  to use spread for props 
<Tippy {...props} content={content} />
```
Popover.propTypes = {
  ...Tippy.propTypes,
  children: childrenPropTypes,
}

```

but we can leave as it is right now, let's just define  `duration` and `delay` in propTypes and defaultProps